apps/dotcom/sync-worker/src/TLDrawDurableObject.ts

```diff
@@ -611,6 +614,8 @@ export class TLDrawDurableObject extends DurableObject {
 		const roomIsReadOnlyForGuests = file.shared && file.sharedLinkType === 'view'
 
 		for (const session of room.getSessions()) {
+			if (file.isDeleted)
+				room.closeSession(session.sessionId, TLSyncErrorCloseEventReason.NOT_FOUND)
```

Comment:
```suggestion
			if (file.isDeleted) {
        room.closeSession(session.sessionId, TLSyncErrorCloseEventReason.NOT_FOUND)
        continue
      }
```

I think we need to do this to prevent the logic below from running